startLumino.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# How it works
+# ./startLumino <ENVIRONMENT_NAME>
+# This script is to start lumino using an environment called <ENVIRONMENT_NAME>.
+# ENVIRONMENT_NAME defaults to luminoNode
+
+# Here some explanation of the parameters:
+# ADDRESS_TO_USE: The address to set to be used by the node
+# PASSWORD_FILE_CONTENT: The password for the private key of the ADDRESS_TO_USE address.
+# TOKEN_NETWORK_REGISTRY: The token network registry contract address to be used by the node.
+# SECRET_REGISTRY: The secret registry contract address to be used by the node.
+# ENDPOINT_REGISTRY: The endpoint registry contract address to be used by the node.
+# HUB_MODE: Sets if the node will be a hub or not
+# RSK_ENDPOINT: The RSK endpoint url
+# EXPOSE_ENDPOINT: The expose url to use by the lumino node
+# PORT: The port number to setup for the lumino node
+# NETWORK_ID: The network id to connect to
+
+ENV_NAME=$1
+
+if [ "" == "${ENV_NAME}" ]; then
+  ENV_NAME=clientEnv
+fi
+
+KETYSTORE_PATH="/home/marcos/rsk/keystore"
+TOKEN_NETWORK_REGISTRY="0x07e1CD1ea2123e3a0f624553761EaD1c1e150CC3"
+SECRET_REGISTRY="0xfA8Bc06C815BD4C4bB641ccb00EBF9CB8BEB2d67"
+ENDPOINT_REGISTRY="0xC14E9A67A87949bC131b9f22a4EB5cd9d9A6728e"
+HUB_MODE=0
+RSK_ENDPOINT=http://localhost:4444
+EXPOSE_ENDPOINT=http://localhost
```

Comment:
i would maybe call this `LUMINO_API_ENDPOINT`

Reply:
done